CONTRIBUTING.md

```diff
@@ -389,6 +392,25 @@ that is not defined in this list below, it will be reviewed for use after you su
 - tracing
 - windows
 
+## Quickstart Preview
+Quickstart Previews are available for contributors to review their new or improved quickstarts directly from the Public I/O site! We provide two ways to view a preview:
+
+### Local Quickstart Preview
+
+- To view a local quickstart preview, you can run the command `yarn preview node-js/express` or `yarn preview catchpoint` using the _path_ to the quickstart.
+- This script needs to be run under the `utils` directory.
+- The script will run a local server for the Public I/O site to fetch files from the specified quickstart.
+- The command line will provide a link that can be navigated to view the quickstart.
+  - In order for local quickstart preview to be enabled on the Public I/O site, there needs to be a `config.yml` file present in the quickstart directory. However, there does not need to have any content inside the file.
```

Comment:
Should we clarify that the `path` param for the yarn command is from the config.yml relative to the quickstart directory? 

Also, should we specify that an additional server needs to be running on a different port from I/O repo?